screens/__tests__/ResultsScreen.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { ResultsScreen, mapStateToProps, mapDispatchToProps } from '../ResultsScreen';
+import { shallow } from 'enzyme';
+import { fetchShows } from '../../thunks/fetchShows';
+
+jest.mock('../../thunks/fetchShows');
+
+describe('ResultsScreen',() => {
+  let wrapper, mockFetchShows, mockShows;
+
+  beforeEach(() => {
+    mockFetchShows = jest.fn();
+    mockShows = [{title: 'new drag'},{title: 'drag this'}]
+    wrapper = shallow(<ResultsScreen
+      shows = {mockShows}
+      fetchShows={mockFetchShows}
+    />);
+  });
+
+  it('should match the snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Great job getting this figured out!! The rest of the components should go pretty smoothly now!